features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/identity.xml.j2

```diff
@@ -3422,4 +3422,9 @@
         <ExtensionTypes>{{extension_mgt.extension_types}}</ExtensionTypes>
     </ExtensionManagementService>
 
+    <!-- IdentityDataStore configurations -->
+    <IdentityDataStore>
```

Comment:
Please check the same file is overridden in asgardeo-product. Check and update that too. repository/resources/conf/templates/repository/conf/identity/identity.xml.j2

Reply:
Ack.